src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/EnclaveDelegate.cs

```diff
@@ -347,28 +337,17 @@ private byte[] ComputeQueryStringHash(string queryString)
             return hash;
         }
 
-        /// <summary>
-        /// Exception when executing a enclave based Always Encrypted query 
-        /// </summary>
         internal class RetriableEnclaveQueryExecutionException : Exception
```

Comment:
This should be Retryable otherwise it read as /re-tree-able/

Reply:
I am afraid that I don't get your comment. Can you explain it more?

Reply:
Trie is wrong, it means something else https://www.dictionary.com/browse/triable 
Ignoring that I can see how you get to trie from try but they aren't homonyms as you can see from the trie data structure etymology https://en.wikipedia.org/wiki/Trie#History_and_etymology . The meaing of this name is that the exception can be re tried and the most appropriate and used form of that is retryable not retrieable.

Even though it already existed it's internal and won't break anything outside this library by changing it. No matter how long it's been spelled as it currently is that time doesn't make it immutable or correct.

Reply:
Fixed